lib/routes/qztc/home/index.ts

```diff
@@ -0,0 +1,112 @@
+import { Data, Route } from '@/types';
+import cache from '@/utils/cache';
+import { parseDate } from '@/utils/parse-date';
+import { load } from 'cheerio';
+import ofetch from '@/utils/ofetch';
+import timezone from '@/utils/timezone';
+
+const rootUrl = 'https://www.qztc.edu.cn/';
+const host = 'www.qztc.edu.cn';
+const name = '泉州师范学院-首页';
+
+export const route: Route = {
+    path: '/home/:type',
+    categories: ['university'],
+    example: '/qztc/home/2093',
+    parameters: { type: '分类，见下表' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: String(name),
```

Comment:
`name` is already a string
https://github.com/DIYgod/RSSHub/blob/63ca61a4450276d53a6e62fe5b233ba26cbc666c/lib/routes/qztc/home/index.ts#L10
No need to use `String()` on a string.